src/managers/ThreadManager.js

```diff
@@ -66,7 +66,7 @@ class ThreadManager extends CachedManager {
    * * `1440` (1 day)
    * * `4320` (3 days) <warn>This is only available when the guild has the `THREE_DAY_THREAD_ARCHIVE` feature.</warn>
    * * `10080` (7 days) <warn>This is only available when the guild has the `SEVEN_DAY_THREAD_ARCHIVE` feature.</warn>
-   * * `'MAX'` Based on the guilds boost count
+   * * `'MAX'` Based on the guild's features
```

Comment:
Why remove this
```suggestion
   * * `'MAX'` Based on the guild's boost level
```

Reply:
It's not *necessarily* the boost count that grants features. They are usually tied, but some guilds may have this feature with even 0 boosts.

Reply:
Yeah only if they’re partnered or verified, in which case boost level 3 is implicit even with no boosts

Reply:
I wouldn't say that is necessarily accurate

the guild feature will exist for private threads if they have private threads, same for 3/7 day times